webapp/javascript/types/flot.d.ts

```diff
@@ -0,0 +1,15 @@
+// we only need this import for its side effects (ie importing flot types)
+// eslint-disable-next-line import/no-unresolved
+import 'flot';
```

Comment:
👍

it might be my mistake (no proofs at this moment), but when I tried to use types from [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/flot/index.d.ts) I found some of "child types" are incorrect, plot returned different values in fact. I hope this attempt goes further

Reply:
well i found that the types are somewhat less strict, but not actually wrong (eg the parameters for the `init` function)

let me know if you find any concrete example and then we can discuss how we can tackle it (contribute upstream, fork it, etc)